src/main/perf/TaskParser.java

```diff
@@ -147,7 +147,7 @@ Task buildQueryTask(String input) throws ParseException {
       Query query = buildQuery(taskType, text, parseMinShouldMatch());
       Query query2 = applyDrillDowns(query, drillDowns);
       Query query3 = applyFilter(query2, filter);
-      return new SearchTask(category, query2, sort, group, topN, doHilite, doStoredLoadsTask, facets, vectorField, doDrillSideways);
+      return new SearchTask(category, query2, sort, group, topN, doHilite, doStoredLoadsTask, facets, null, doDrillSideways);
```

Comment:
I removed this since otherwise we would be adding vector load to every existing task, and would see an across-the-board penalty